src/components/v2/Packages/PackagePanel.js

```diff
@@ -41,7 +42,7 @@ class PackagePanel extends Component {
 	                            <div><a onClick={this.handleAttachmentClick}>{this.props.uploadPackage.attachments.length} attachment(s)</a></div>
 	                            <div><a onClick={this.handleMetadataClick}>Show package metadata</a></div>
 	                            <div>
-	                                <Button className="btn btn-primary">
```

Comment:
Let's not set a baseURL like this.  I believe it will cause issues when we deploy (since it will likely try to go to the user's localhost) and it will likely cause issues when we go to Apache.  If you use /api/{uri}, then nginx is set up to know to pass this to our back-end.